src/local.c

```diff
@@ -56,7 +56,10 @@ check_localdate(void)
         /* *INDENT-OFF* */
         fprintf(stdout, "%s", ANSI_BOLD_ON);
         fprintf(stdout, "%s", ANSI_COLOR_CODE_FG);
-        fprintf(stdout, "Local data is older than two weeks, use --update to update it.\n\n");
+        fprintf(stdout, "Local data is older than two weeks, attempting to automatically update it...\n");
```

Comment:
```suggestion
        fprintf(stdout, "Local data is older than two weeks, attempting to update it...\n");
```

No need for the word automatic here, the user knows it's automatic because it was automatically done for them!